imports/plugins/included/payments-paystack/server/methods/paystack-payment-methods.app-test.js

```diff
@@ -0,0 +1,240 @@
+
+  beforeEach(function () {
+    sandbox = sinon.sandbox.create();
+  });
+
+  afterEach(function () {
+    sandbox.restore();
+  });
+
+  it("should return data from ThirdPartyAPI authorize", function () {
+    const cardData = {
+      name: "Test User",
+      email: "test@test.com"
+    };
+    const paymentData = {
+      currency: "USD",
+      total: "19.99"
+    };
+
+    const transactionType = "authorize";
+    const transaction = PaystackApi.methods.authorize.call({
+      transactionType: transactionType,
+      cardData: cardData,
+      paymentData: paymentData
+    });
+    expect(transaction).to.not.be.undefined;
+  });
+
+  it("should return risk status for flagged test card", function () {
+    const cardData = {
+      name: "Test User",
+      email: "test@test.com"
+    };
+    const paymentData = {
+      currency: "USD",
+      total: "19.99"
+    };
+
+    const transactionType = "authorize";
+    const transaction = PaystackApi.methods.authorize.call({
+      transactionType: transactionType,
+      cardData: cardData,
+      paymentData: paymentData
+    });
+
+    expect(transaction.riskStatus).to.be.defined;
+  });
+
+  it("should return data from ThirdPartAPI capture", function (done) {
+    const authorizationId = "abc123";
+    const amount = 19.99;
+    const results = PaystackApi.methods.capture.call({
+      authorizationId: authorizationId,
+      amount: amount
+    });
+    expect(results).to.not.be.undefined;
+    done();
+  });
+});
+
+
+describe("Submit payment", function () {
+  let sandbox;
+
+  beforeEach(function () {
+    sandbox = sinon.sandbox.create();
+  });
+
+  afterEach(function () {
+    sandbox.restore();
+  });
+
+  it("should call Paystack API with card and payment data", function () {
+    // this is a ridiculous timeout for a test that should run in subseconds
+    // but a bug in the Meteor test runner (or something) seems to make this test stall
+    // it actually stalls after the entire test is completed
+    this.timeout(30000);
+    const cardData = {
+      name: "Test User",
+      email: "test@test.com"
+    };
+    const paymentData = {
+      currency: "USD",
+      total: "19.99"
+    };
+
+    const authorizeResult = {
+      saved: true,
+      currency: "USD"
+    };
+
+    const authorizeStub = sandbox.stub(PaystackApi.methods.authorize, "call", () => authorizeResult);
+    const results = Meteor.call("paystackSubmit", "authorize", cardData, paymentData);
+    expect(authorizeStub).to.have.been.calledWith({
+      transactionType: "authorize",
+      cardData: cardData,
+      paymentData: paymentData
+    });
+    expect(results.saved).to.be.true;
+  });
+
+  it("should throw an error if card data is not correct", function () {
+    const badCardData = {
+      name: "Test User",
+      email: "test@test.com"
+    };
+
+    const paymentData = {
+      currency: "USD",
+      total: "19.99"
+    };
+
+    // Notice how you need to wrap this call in another function
```

Comment:
This comment does not describe what is happening here. Please make this comment more descriptive or remove.